docs/standard/threading/threads-and-threading.md

```diff
@@ -1,91 +1,50 @@
-  
- Resources that require synchronization include:  
-  
--   System resources (such as communications ports).  
-  
--   Resources shared by multiple processes (such as file handles).  
-  
--   The resources of a single application domain (such as global, static, and instance fields) accessed by multiple threads.  
-  
-### Threading and Application Design  
- In general, using the <xref:System.Threading.ThreadPool> class is the easiest way to handle multiple threads for relatively short tasks that will not block other threads and when you do not expect any particular scheduling of the tasks. However, there are a number of reasons to create your own threads:  
-  
--   If you need a task to have a particular priority.  
-  
--   If you have a task that might run a long time (and therefore block other tasks).  
-  
--   If you need to place threads into a single-threaded apartment (all **ThreadPool** threads are in the multithreaded apartment).  
-  
--   If you need a stable identity associated with the thread. For example, you should use a dedicated thread to abort that thread, suspend it, or discover it by name.  
-  
--   If you need to run background threads that interact with the user interface, the .NET Framework version 2.0 provides a <xref:System.ComponentModel.BackgroundWorker> component that communicates using events, with cross-thread marshaling to the user-interface thread.  
-  
-### Threading and Exceptions  
- Do handle exceptions in threads. Unhandled exceptions in threads, even background threads, generally terminate the process. There are three exceptions to this rule:  
-  
--   A <xref:System.Threading.ThreadAbortException> is thrown in a thread because <xref:System.Threading.Thread.Abort%2A> was called.  
-  
--   An <xref:System.AppDomainUnloadedException> is thrown in a thread because the application domain is being unloaded.  
-  
--   The common language runtime or a host process terminates the thread.  
-  
- For more information, see [Exceptions in Managed Threads](../../../docs/standard/threading/exceptions-in-managed-threads.md).  
-  
+# Threads and threading
+
+Multithreading allows you to increase the responsiveness of your application and, if your application runs on a multiprocessor or multi-core system, increase its throughput.
+
+## Processes and threads
+
+A *process* is an executing program. An operating system uses processes to separate the applications that are being executed. A *thread* is the basic unit to which an operating system allocates processor time. Multiple threads can run in the context of a process. All threads of a process share its virtual address space. A thread can execute any part of the program code, including parts currently being executed by another thread.
```

Comment:
I think some of the basic information on what a thread is should be added back before the "Multiple threads..." sentence. Particularly: "Each thread maintains exception handlers, a scheduling priority, and a set of structures the system uses to save the thread context until it is scheduled. The thread context includes all the information the thread needs to seamlessly resume execution, including the thread's set of CPU registers and stack, in the address space of the thread's host process."

Reply:
@rpetrusha I've put those sentences back, rephrased and with some information dropped. However, I hope that the essence is there. Please let me know, if it's not true and I'll come up with better phrasing.